src/client/state/cons.js

```diff
@@ -47,7 +47,7 @@ const cons = {
       CLICK_REPLY_TO: 'CLICK_REPLY_TO',
       OPEN_SEARCH: 'OPEN_SEARCH',
       OPEN_REUSABLE_CONTEXT_MENU: 'OPEN_REUSABLE_CONTEXT_MENU',
-      OPEN_NAVIGATION: 'OPEN_NAVIGATION',
+      NAVIGATION_OPENED: 'NAVIGATION_OPENED',
```

Comment:
This can be as it was before (OPEN_NAVIGATION) because it is an action that user calls. Other one was event (already has happened). It's like user say OPEN_NAVIGATION and system open it and inform view NAVIGATION_OPENED.